msal4j-brokers/src/main/java/com/microsoft/aad/msal4jbrokers/MsalRuntimeBroker.java

```diff
@@ -3,15 +3,7 @@
 
 package com.microsoft.aad.msal4jbrokers;
 
```

Comment:
Note that msal_runtime authResult will return an unsigned token. This is the correct way to get the token. @MSamWils - I think this is an msal_runtime bug.

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/9e4bf560970fb5be9a72b83b3a446ce49c545576/src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs#L325

Reply:
After trying a couple options it seems cleaner to handle this in the interop package rather than this package. 

Basically, if this is some MSALRuntime-specific behavior then the interop should deal with it, and the msal4j-brokers package should just expect a valid result based on the parameters it sends to the interop. I'll take care of this in a PR over in the CPP repo.